cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -236,7 +234,9 @@ public void drop(String name) throws Exception {
 
   @Override
   public void upgrade(String tableName, Properties properties) throws Exception {
```

Comment:
Upgrade should not takes table name. It should just take a namespace and the upgrade should performs upgrade all queue tables under that namespace.